contracts/CEther.sol

```diff
@@ -114,6 +114,15 @@ contract CEther is CToken {
         requireNoError(err, "mint failed");
     }
 
+    /**
+     * @notice A public function to sweep accidental ERC-20 transfers to this contract. Tokens are sent to admin (timelock)
+     * @param token The address of the ERC-20 token to sweep
+     */
+    function sweepToken(EIP20NonStandardInterface token) external {
```

Comment:
not sure about merging the cEth change in since it won't actually be receiving the upgrade 🤔, wdyt?

Reply:
Yeah, you are probably right. Should try to keep GitHub as close to chain as possible. I just thought to do it now as any future cETH implementation should have this functionality.

Reply:
I'm of the mindset that if a future deploy will FOR SURE contain it, include it. This would be an argument for including cETH.